documentation/modules/proc-iot-creating-device.adoc

```diff
@@ -11,29 +11,45 @@ After installing the IoT services and creating an IoT project, you can create an
 ifeval::["{cmdcli}" == "oc"]
 :registry: $(oc -n enmasse-infra get routes device-registry --template='{{ .spec.host }}')
 endif::[]
-ifeval::["{cmdcli}" == "kubectl"]
+ifeval::["{cmdcli}" != "oc"]
 :registry: $(kubectl -n enmasse-infra get service iot-device-registry-external -o jsonpath={.status.loadBalancer.ingress[0].hostname}):31443
 endif::[]
 
 .Prerequisites
-* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#installing-services-messaging-iot[The IoT services are installed].
-* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-creating-project-messaging-iot[An IoT project is created].
-
-.Procedure
-
-. Export the device registry host:
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#installing-services-{context}[The IoT services are installed].
```

Comment:
For me the `{context}` variable gets properly expanded to `messaging-iot` when I generate the documentation. As the `{context}` variable is used in the ID itself, I assumed that this would ensure the links don't break you someone changes the value of the `{context}` variable.